pkgs/os-specific/linux/udisks/2-default.nix

```diff
@@ -38,6 +39,11 @@ stdenv.mkDerivation rec {
     "--with-udevdir=$(out)/lib/udev"
   ];
 
+  makeFlags = [
+    "INTROSPECTION_GIRDIR=$(out)/share/gir-1.0"
```

Comment:
The GIR file is moved in `gobjectIntrospection` setup hook but we should still use `dev` instead of `out` here.

Reply:
Ah, missed that one. Should be fixed now.